index.js

```diff
@@ -6,9 +6,16 @@ var TestTransformFilter = require('./lib/tests-transform-filter');
 module.exports = {
   name: 'ember-qunit-nice-errors',
 
+  included: function(app) {
+    this._super.included.apply(this, arguments);
+
+    this.addonConfig =
+      app.project.config(app.env)['ember-qunit-nice-errors'] || {};
```

Comment:
There's a known issue we need cover from

See https://github.com/ember-cli/ember-cli/issues/3718
and a workaround https://github.com/samselikoff/ember-cli-mirage/blob/master/index.js#L24-L27